packages/rspack/etc/api.md

```diff
@@ -5928,7 +5928,7 @@ export interface LoaderContext<OptionsType = {}> {
     // (undocumented)
     getOptions(schema?: any): OptionsType;
     // (undocumented)
-    getResolve(options: Resolve): (context: any, request: any, callback: any) => Promise<any>;
+    getResolve(options: Resolve): (context: any, request: any, callback: any) => Promise<any> | undefined;
```

Comment:
should we align with webpack?

![image](https://github.com/user-attachments/assets/f41aacac-17c9-4d75-b42d-158c76836ec9)

Reply:
okay, in fact, it does have the possibility of returning undefined..